easybuild/easyconfigs/c/ccache/ccache-4.6.1-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# Updated to 4.6.1: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'ccache'
+version = '4.6.1'
+
+homepage = 'https://ccache.dev/'
+description = """Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by
+caching previous compilations and detecting when the same compilation is being done again"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/ccache/ccache/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['59b28a57c3a45e48d6049001999c9f94cd4d3e9b0196994bed9a6a7437ffa3bc']
+
+osdependencies = [('glibc-static', 'libc6-dev', 'glibc-devel')]
+
+builddependencies = [
```

Comment:
@sassy-crick You will also need to add `binutils` as a build dep here (since you're using `GCCcore` as a toolchain)

Reply:
fixed in e14b03f